lib/grape_entity/entity.rb

```diff
@@ -454,16 +454,17 @@ def should_return_attribute?(attribute, options)
     end
 
     def only_fields(options, for_attribute = nil)
-      return nil unless options[:only]
+      return unless options[:only]
 
       @only_fields ||= options[:only].each_with_object({}) do |attribute, allowed_fields|
         if attribute.is_a?(Hash)
           attribute.each do |attr, nested_attrs|
+            attr = attr.to_sym
```

Comment:
I think you should convert those to strings. Until Ruby 2.2 symbols are not garbage collected so you're creating an opportunity for some DDoS if this is used with user-supplied data.